core-bundle/src/Cache/ContaoCacheWarmer.php

```diff
@@ -162,7 +162,7 @@ private function generateLanguageCache(string $cacheDir): void
                 $subfiles = $this->finder
                     ->findIn(Path::join('languages', $language))
                     ->files()
-                    ->name('/^'.$name.'\.(php|xlf)$/')
+                    ->name("/^$name\\.(php|xlf)$/")
```

Comment:
Why did you add another `\`? Wouldn't that actually match a backslash now?

Reply:
It wasn't me. The `EscapeImplicitBackslashesFixer` wanted it this way. 😄

Reply:
Isn't this the typical "why you need `\\\\` within double quotes to actually have one backslash" case?

Reply:
Yeah, I guess you're right.

Reply:
Although, to be honest - I do not fully understand why this is the case...

When I try it with `preg_match` it's the same outcome, but when I try it e.g. on https://regex101.com `\\.` actually matches a literal `\` + any character while `\.` matches the literal `.` (which is what it should be).

Reply:
https://3v4l.org/h2Og9

Reply:
@m-vo `"\\."` is a php string, not a RegExp directly. 
So the PHP string `"\\."` results in the RegExp `\.` which results in the literal string `.` to match.